client/packages/host/src/CommandK.tsx

```diff
@@ -256,11 +256,23 @@ export const CommandK: FC<PropsWithChildrenOnly> = ({ children }) => {
   }
 
   if (store?.storeMode === StoreModeNodeType.Dispensary) {
+    actions.push({
+      id: 'navigation:prescription',
+      name: `${t('cmdk.goto-prescriptions')} (p)`,
+      keywords: 'prescription',
+      shortcut: ['p'],
+      perform: () =>
+        navigate(
+          RouteBuilder.create(AppRoute.Dispensary)
+            .addPart(AppRoute.Prescription)
+            .build()
+        ),
+    });
     actions.push({
       id: 'navigation:patients',
-      name: `${t('cmdk.goto-patients')} (p)`,
+      name: `${t('cmdk.goto-patients')} (g+p)`,
       keywords: 'patient',
-      shortcut: ['p'],
+      shortcut: ['g', 'p'],
```

Comment:
Changed to multi-key since prescription seems like it would be used more often than patient page..